spec/refile/spec_helper.rb

```diff
@@ -38,7 +38,8 @@ def presign
   File.open(tempfile.path, "r")
 end
 
-Refile.processor(:convert_case) do |file, format:|
+Refile.processor(:convert_case) do |file, options = {}|
+  format = options.fetch :format
```

Comment:
Pretty sure they have a different parser all together. For some reason the format was always coming through as nil. This is likely one of the things that will be ironed out before JRuby 9k drops, but doesn't work today.